tests/foreman/cli/test_satellitesync.py

```diff
@@ -64,13 +64,58 @@ def config_export_import_settings():
     Settings.set({'name': 'content_disconnected', 'value': disconnected_mode_value})
 
 
+@pytest.fixture(scope='function')
+def export_cleanup_function(default_sat, function_org):
+    """Deletes export/import dirs of function org"""
+    yield
+    # Deletes directories created for export/import test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}').stdout == ''
+    assert default_sat.execute(f'rm -rf {IMPORT_DIR}/{function_org.name}').stdout == ''
+
+
+@pytest.fixture(scope='function')
+def export_cleanup_module(default_sat, module_org):
+    """Deletes export/import dirs of module_org"""
+    yield
+    # Deletes directories created for export/import test
+    assert default_sat.execute(f'rm -rf {EXPORT_DIR}/{module_org.name}').stdout == ''
+    assert default_sat.execute(f'rm -rf {IMPORT_DIR}/{module_org.name}').stdout == ''
+
+
+def validate_filepath(sat_obj, org):
+    """Checks the existence of certain files in a dir"""
+    result = sat_obj.execute(
+        fr'find {EXPORT_DIR}{org.name} -type f \( -name "*.json" -o -name "*.tar.gz" \)'
+    )
+    return result.stdout
+
+
+def move_pulp_archive(sat_obj, org, export_message):
+    """
+    Moves exported archive(s) and its metadata into import directory,
+    sets ownership, returns import path
+    """
+    sat_obj.execute(f'mv {EXPORT_DIR}/{org.name} {IMPORT_DIR}')
+    sat_obj.execute(f'chown -R pulp:pulp {IMPORT_DIR}')
+
+    # removes averything before export path,
+    # replaces EXPORT_PATH by IMPORT_PATH,
+    # removes metadata filename
+    import_path = os.path.dirname(
+        IMPORT_DIR
```

Comment:
For readability and to avoid the E203, I'd create a variable here

Secondly I'd advise against using `+` for string concatenation, especially since we don't type hint, use `f-string` instead. 

Third, did you check whether you can make this replacement reliably with `replace` instead of directly trying to slice and concatenate? Or better yet, with `Pathlib` tools like `relative_to`, `join_path`, or `parts`?

https://docs.python.org/3/library/pathlib.html

Reply:
This is a bit more complicated... We don't receive just path to archive, but need to extract it from an export message, which looks like this:
`Generated /export_dir/org_name/and_some_more_subdirectories/metadata.json`
So in that function we move the exported archive and metadata into import directory while keeping the subdirectory structure, then we return the import path (without metadata filename).
I think it could be more readable with `re`, so I suggest this change:
`import_path = os.path.dirname(re.sub(rf'.*{EXPORT_DIR}', IMPORT_DIR, export_message))`

Reply:
@lhellebr I like that re.sub approach as well, thanks for explaining!

Reply:
implemented in latest commit